MoviePlay/MoviePlay/API/Request/Urls.swift

```diff
@@ -9,27 +9,29 @@
 import Foundation
 
 struct URLs {
-    private static var aPIBaseUrl = "https://api.themoviedb.org/3"
+    private static var apiBaseUrl = "https://api.themoviedb.org/3"
     
-    public static var urlYoutobe = "https://www.youtube.com/embed"
+    public static var youtobeURL = "https://www.youtube.com/embed"
```

Comment:
vẫn sai. Cái này anh thấy hình như 2-3 cái comment kêu sửa rồi mà em cứ sửa đi sửa lại là sai vậy. 
```suggestion
    public static var youtubeURL = "https://www.youtube.com/embed"
```

Reply:
youtube nhe..:v